src/DGtal/base/CowPtr.h

```diff
@@ -75,9 +75,35 @@ namespace DGtal
     // no need for ~CowPtr - the CountedPtr takes care of everything.
     CowPtr(const CowPtr& r) noexcept             : myPtr(r.myPtr) {}
     /**
-     * @todo JOL: check this.
+       Builds a copy-on-write pointer from a counted pointer. Requires
+       an extra dummy parameter in order to solve ambiguities when
+       casting from Clone<T> to CowPtr<T>.
+       
+       @param r any counted pointer
+       
+       @note Since 1.0, there is no more the direct constructor
+       CowPtr<T>::CowPtr( CountedPtr<T> ). Indeed, it was creating an
+       ambiguity when using conversion operator in class
+       Clone<T>. More precisely it was impossible to have two
+       conversion operators: Clone<T>::operator CountedPtr<T> and
+       Clone<T>::operator CowPtr<T> since this was creating a
+       compilation ambiguity. This is why there is now a dummy bool
+       parameter in this constructor. And there are the conversion
+       operator Clone<T>::operator CountedPtr<T> and
+       Clone<T>::operator CowPtr<T>.
```

Comment:
Redundant sentence?

Reply:
I have rephrased it.

Reply:
Thx, just a typo at operator (see other comment)